### PR TITLE
[EASY] lowercase queries and cleanup leftover correction

### PR DESCRIPTION
**Pull request by harisang**
This PR switches the convention for sql queries to always use lowercase for keywords and also removes outdated fix.

### CODE REVIEW

File: queries/orderbook/barn_batch_rewards.sql [hunk 1]
```diff
@@ -212,63 +212,63 @@ reward_per_auction AS (
...
+    from primary_rewards pr
+        left outer join aggregate_partner_fees_per_solver aif on pr.solver = aif.solver
 ) --
-SELECT
+select
     *
-FROM
+from
     aggregate_results
 ORDER BY
```
Comment (harisang) [inline queries/orderbook/barn_batch_rewards.sql]:
```suggestion
order by
```

File: queries/orderbook/prod_batch_rewards.sql [hunk 2]
```diff
@@ -215,63 +212,63 @@ reward_per_auction AS (
...
+    from primary_rewards pr
+        left outer join aggregate_partner_fees_per_solver aif on pr.solver = aif.solver
 ) --
-SELECT
+select
     *
-FROM
+from
     aggregate_results
 ORDER BY
```
Comment (harisang) [inline queries/orderbook/prod_batch_rewards.sql]:
```suggestion
order by
```

**Review by fhenneke — APPROVED** [review]
Is this compatible with sqlfluff? Might be an alternative to fix the style of all queries.

File: queries/.sqlfluff [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[sqlfluff]
+dialect = ansi
+exclude_rules = L046, RF02, RF06, CP02
+max_line_length = 0
```
Comment (fhenneke) [inline queries/.sqlfluff]:
```suggestion
max_line_length = 0

```

**Pull request merged**